### ISSUE TITLE
Drop WordPress template engine

### ISSUE THREAD

**Issue by haroldangenent** [trendwerk/trendpress#706]
This is a proposal for dropping the WordPress template inclusion. Right now, variables can not be injected into the templates, because of WordPress' template engine (which you can find in `template-loader.php`).

We might be able to get rid of most controllers we have now and we could run the entire theme from one entry point and inject everything we need intro our controllers and views.

**Comment by luukdv** [comment]
Very interesting. Do you want to give this a go in a separate branch sometime?

**Comment by haroldangenent** [comment]
Yes, and the heavy work would probably be done by a separate library.

**Comment by haroldangenent** [comment]
Discussed this today, decided to create a separate library that should be implemented by TrendPress. I'll keep this issue open for now, but removing the discussion label.
